<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QPushButton" name="resampleBtn">
        <property name="text">
         <string>开始 重采样(Resample)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="recordAudioBtn">
        <property name="text">
         <string>开始 录制 pcm</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="yuvRecordBtn">
        <property name="text">
         <string>开始 录制 yuv</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="aacDecodeBtn">
        <property name="text">
         <string>开始 aac 文件解码(decode)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="playWavBtn">
        <property name="text">
         <string>开始 播放 wav</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="playYuvBtn">
        <property name="text">
         <string>开始 使用 SDL 播放 yuv</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="encodeH264Btn">
        <property name="text">
         <string>开始 h264 编码(encode)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="playAudioBtn">
        <property name="text">
         <string>开始 播放 pcm</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="switchButton">
        <property name="text">
         <string>开始 pcm to wav</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="aacEncodeBtn">
        <property name="text">
         <string>开始 aac 文件编码(encode)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="recordWavBtn">
        <property name="text">
         <string>开始 录制 wav</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="playYuvByQtBtn">
        <property name="text">
         <string>开始 使用Qt原生播放 yuv</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="decodeH264Btn">
        <property name="text">
         <string>开始 h264 解码(decode)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="demuxerBtn">
        <property name="text">
         <string>开始 解封装(demuxer)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">max-height:100px;
font-size: 30px;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="titleLabel">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>时间:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="timeLabel">
            <property name="minimumSize">
             <size>
              <width>374</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>00:00:00</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>命令行1：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cLineLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="copyBtn">
           <property name="text">
            <string>copy</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>命令行2：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cLineLabel_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="copyBtn_2">
           <property name="text">
            <string>copy</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
